src/components/dialog/dialog/dialog.ts

```diff
@@ -233,10 +234,32 @@ export class SbbDialogElement extends SbbNegativeMixin(LitElement) {
     }
   }
 
+  protected override firstUpdated(): void {
+    this._ariaLiveRef = this.shadowRoot!.querySelector('.sbb-screen-reader-only')!;
+    this._dialogTitleElement = this.querySelector('sbb-dialog-title')!;
```

Comment:
I added this because we would see a flash on the first open if it's negative, so I had to sync it earlier.

Reply:
I need this to sync the negative state before the first open. I could remove it from `open()`, even though the title might change on the next opens and keeping it can be more stable (?) You tell me 👍

Reply:
just keep it and add a comment :)